src/snowcli/cli/snowpark/__init__.py

```diff
@@ -0,0 +1,11 @@
+import typer
+
+from snowcli.cli.snowpark.function import app as function_app
+from snowcli.cli.snowpark.package import app as package_app
+from snowcli.cli.snowpark.procedure import app as procedure_app
+
+app = typer.Typer(context_settings={"help_option_names": ["-h", "--help"]})
+
+app.add_typer(function_app, name="function")
```

Comment:
```suggestion
app.add_typer(function_app)
```
Will this work? The name will be taken from type instance